File: Easy/Bank-robbers/Bank-robbers.swift
let R = Int(readLine()!)!, V = Int(readLine()!)!
var rs = Array(repeating: 0, count: R)
for i in 0..<V {
    let I = readLine()!.split(separator: " ").map{Int($0)!}
    rs[0] += [125,625,3125,15625,78125,390625][I[0] - 3] << I[1]
    rs.sort()
}
print(rs.last!)

File: Easy/Bank-robbers/Bank-robbers.js
let RS = Array(+readline()).fill(0)
for (let i = readline(); i--; RS.sort((a, b) => a - b)) {
    let I = readline().split(' ')
    RS[0] += [125, 625, 3125, 15625, 78125, 390625][I[0] - 3] << I[1]
}
console.log(Math.max(...RS));
